### ISSUE TITLE
Windows Compilation Error

### ISSUE THREAD

**Issue by sdogruyol** [crystal-loot/exception_page#36]
When running the simplest Kemal app possible on Windows using Crystal 1.8.0

I got the following coming from `exception_page`

```
In lib\exception_page\src/exception_page/exception_page.ecr:14:6

 14 | <%-
           ^
Error: expected '\n' after '\r'
```

**Comment by mamantoha** [comment]
@sdogruyol https://github.com/crystal-loot/exception_page/pull/37

I believe this is not a complete solution, but it will enable the CI to run the specs.

**Comment by jwoertink** [comment]
Ok, merged that in. If y'all want to pull from master and test and let me know if it works, I'll cut a new release.

**Comment by zw963** [comment]
> Ok, merged that in. If y'all want to pull from master and test and let me know if it works, I'll cut a new release.

It works. check my github action.

https://github.com/crystal-china/crystal_hello_world/actions/runs/4908312756/jobs/8764005993

**Comment by zw963** [comment]
so, @sdogruyol, need update kemal dependencies to use new exception_page after it release new version, right?

**Comment by straight-shoota** [comment]
This was a compiler bug and should be fixed in master (and the upcoming Crystal 1.9 release).
https://github.com/crystal-lang/crystal/issues/13484
